servers/rendering/shader_language.h

```diff
@@ -848,9 +882,13 @@ class ShaderLanguage {
 		IDENTIFIER_LOCAL_VAR,
 		IDENTIFIER_BUILTIN_VAR,
 		IDENTIFIER_CONSTANT,
-	};
+		IDENTIFIER_MAX,
+	} last_type = IDENTIFIER_MAX;
```

Comment:
It's equivalent but I think it looks clearer if `last_type` is defined separately, since it's a named enum:
```
IdentifierType last_type = IDENTIFIER_MAX;
```

Reply:
Fixed